### PR TITLE
Fix minor typo in `cloneWithProps` doc

### PR DESCRIPTION
**Pull request by sumbach**

### CODE REVIEW

**Comment by sumbach** [comment]
CLA submitted (though I don't think this contribution is significant enough to warrant copyright protection).

File: docs/docs/10.5-clone-with-props.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ prev: test-utils.html
 next: create-fragment.html
 ---
 
-In rare situations a element may want to change the props of a element that it doesn't own (like changing the `className` of a element passed as `this.props.children`). Other times it may want to make multiple copies of a element passed to it. `cloneWithProps()` makes this possible.
+In rare situations, an element may want to change the props of an element that it doesn't own (like changing the `className` of a element passed as `this.props.children`). Other times it may want to make multiple copies of a element passed to it. `cloneWithProps()` makes this possible.
```
Comment (jdreesen) [inline docs/docs/10.5-clone-with-props.md]: There are two more occurences of `a` which should be `an` instead.

**Comment by jsfb** [comment]
Looks like this was fixed in https://github.com/facebook/react/pull/3589, closing.

**Comment by sumbach** [comment]
Agreed! Thanks for following up, @jsfb :smile:

**Pull request closed without merging**